windows/src/desktop/kmshell/main/Keyman.System.UpdateStateMachine.pas

```diff
@@ -987,32 +976,27 @@ procedure InstallingState.LaunchInstallPackageProcess;
 begin
   if not kmcom.SystemInfo.IsAdministrator then
   begin
-    KL.Log('InstallingState.LaunchInstallPackageProcess not IsAdmin');
     if CanElevate then
     begin
-      KL.Log('InstallingState.LaunchInstallPackageProcess CanElevate');
       executeResult := WaitForElevatedConfiguration(0, '-ikp');
       if (executeResult <> 0) then
       begin
         TKeymanSentryClient.Client.MessageEvent
           (Sentry.Client.SENTRY_LEVEL_ERROR,
           'Executing kmshell process to install keyboard packages failed:"' +
           IntToStr(Ord(executeResult)) + '"');
-        KL.Log('InstallingState.LaunchInstallPackageProcess Error elevating');
         ChangeState(IdleState);
       end;
     end
     else
     begin
-      KL.Log('InstallingState.LaunchInstallPackageProcess require user with admin');
       // TODO: epic-windows-updates How do we alert the user that package requires a user with admin rights
       // ShowMessage('Some of these updates require an Administrator to complete installation.  Please login as an Administrator and re-run the update.');
```

Comment:
@mcdurdin Not sure if we can use ShowMessage here.

Reply:
No, we cannot pop UI here